basics/hello-world/README.md

```diff
@@ -103,22 +128,29 @@ of the `JupyterFrontEndPlugin` class:
 
 <!-- prettier-ignore-start -->
 ```ts
-// src/index.ts#L9-L12
+// src/index.ts#L9-L18
 
 const extension: JupyterFrontEndPlugin<void> = {
   id: 'hello-world',
   autoStart: true,
   activate: (app: JupyterFrontEnd) => {
+    console.log('the JupyterLab main application:', app);
+  }
+};
+
+export default extension;
+
 ```
 
 ```ts
-// src/index.ts#L13-L13
-
-console.log('the JupyterLab main application:', app);
+    console.log('JupyterLab extension hello-world is activated!');
 ```
 
 ```ts
-// src/index.ts#L17-L17
+// src/index.ts#L14-L17
+
+  }
```

Comment:
Looks like this might require a `embedme` update?

Reply:
Thanks for the review.

I corrected the snippet before this one. From the text, the goal is to present the code state generated from the template. So this is split in 3 snippets as the code changes latter in the example.